sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -696,6 +694,7 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
         "offering.submittedDate",
         "offering.courseLoad",
         "offering.offeringStatus",
+        "offering.precedingOffering.id",
```

Comment:
Not sure if it is working but usually we reference the alias directly like "precedingOffering.id".

Reply:
Yes i believe we can directly use the precedingOffering.id rather going through the offering.

https://github.com/bcgov/SIMS/blob/672f0a4ab89c89cf945cb198f4e778849a182ff5/sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering.service.ts#L902-L916

Reply:
Changed it to `precedingOffering.id`.